src/objects/token/admin.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib import admin
+
+from .models import Permission, TokenAuth
+
+
+class PermissionInline(admin.TabularInline):
+    model = Permission
+    extra = 1
+
+
+@admin.register(TokenAuth)
+class TokenAuthAdmin(admin.ModelAdmin):
+    list_display = ("token", "contact_person", "created")
```

Comment:
I think the admin should just generate a token, and not be able to change it afterwards (readonly field).